devito/arch/archinfo.py

```diff
@@ -571,6 +571,11 @@ def memavail(self, deviceid=0):
         except (AttributeError, KeyError):
             return None
 
+class IntelDevice(Device):
```

Comment:
Before I trigger CI -- classes require two blank lines before and after their definition, so this needs two more blank lines, one here and one at the bottom, or it will fail the flake8 build